src/wmnds/patterns/content-cards/inline-navigation/_inline-navgiation.scss

```diff
@@ -0,0 +1,9 @@
+.wmnds-content-card {
+  &__list {
+    // Current page
+    [aria-current="page"] {
+      padding-left: $size-xsm;
+      border-left: 3px solid get-color(information);
```

Comment:
@daylesalmon in this case, should we round it to 4px as it is our closest global measure or should we keep it like it is?

Reply:
I've just confirmed with designers that this can be removed (the purple line with "overview" next to it)
![image](https://user-images.githubusercontent.com/13015201/92104527-594b1380-edd9-11ea-9180-5a48bb2ea02d.png)

This text and border can be replaced with the Inset Text component: https://designsystem.wmnetwork.co.uk/components/inset-text/